tests/test_opds.py

```diff
@@ -2050,6 +2050,8 @@ def title(x):
         # the breadcrumbs will be start at that lane -- we won't see
         # the sublane's parent or the library root.
         sublane.root_for_patron_type = ["ya"]
+        assert_breadcrumbs([], sublane)
+
```

Comment:
This is the test that was missing. Previously hitting a patron root lane would give you an "All books" breadcrumb when it should have given you no breadcrumbs at all.